ncc/examples/monogram.c

```diff
@@ -0,0 +1,3536 @@
+#include <uvm/syscalls.h>
+#include <uvm/utils.h>
+
+
+// Monogram 12x7 pixel font
+
+// > # CREDITS
+// > 
+// > Monogram is a free and Creative Commons Zero pixel font,
+// > made by Vinícius Menézio (@vmenezio).
+// > 
+// > https://datagoblin.itch.io/monogram
+// > 
+// > 
+// > # SPECIAL THANKS
+// > 
+// > thanks to Ateş Göral (@atesgoral) for creating the bitmap font converter:
+// > https://codepen.io/atesgoral/details/RwGOvPZ
+// > 
+// > thanks to Éric Araujo (@merwok_) for the inital port of monogram to PICO-8:
+// > https://itch.io/post/2625522
+
+u64 font_monogram_number_of_characters = 390;
+u64 font_monogram_height = 12;
+u64 font_monogram_width = 7;
+
+u8 font_monogram_data[390][12];
```

Comment:
This is 12 bytes of data per char, as in 12 rows, and each byte represents the pixels in that row? Would be good to explain in a comment.